katz-rx/src/main/kotlin/katz/rx/observable/instances/ObservableWrap.kt

```diff
@@ -0,0 +1,19 @@
+package katz
+
+import io.reactivex.Observable
+
+typealias ObservableWrapKind<A> = HK<ObservableWrap.F, A>
+
+fun <A> ObservableWrapKind<A>.ev(): ObservableWrap<A> = this as ObservableWrap<A>
+fun <A> ObservableWrapKind<A>.value(): Observable<A> = this.ev().value
+
+data class ObservableWrap<A>(val value: Observable<A>) : ObservableWrapKind<A> {
```

Comment:
i don't like to much `ObservableWrap` always come to my mind a burrito xD
I propose `ObservableHK`or `ObservableKindAdapter` or some variation of HK with suffix Adapter instead of Wrap.

this comment it's not mandatory, just wanna open de discussion.